src/pngtocss.c

```diff
@@ -525,10 +546,10 @@ int main(int argc, char **argv)
 	}
 
 	for(i=1; i<argc; i++) {
-		if((err = process_file(argv[i])) != OK) {
+		if((err = process_file(argv[i])) != OK && err != E_FILE) {
 			print_error(argv[i], err);
 		}
 	}
 
 	return err;
-}
+}
```

Comment:
make sure there's a newline at the end of the file to keep with the unix text file convention.